mappings/net/minecraft/entity/ItemSteerable.mapping

```diff
@@ -10,6 +10,7 @@ CLASS net/minecraft/class_4981 net/minecraft/entity/ItemSteerable
 		ARG 4 childRideable
 	METHOD method_26315 setMovementInput (Lnet/minecraft/class_243;)V
 		ARG 1 movementInput
+	METHOD method_26316 getBoostedSpeed ()F
```

Comment:
Not boosted, but like steered speed. A multiplier for boosting is applied later. See also #1230